### PR TITLE
FEAT: detailed card completed with correction based on previous issue #1

### PR DESCRIPTION
**Pull request by gami34**
What this PR does (required):
all corrections raised during issue #1 were corrected and implemented
API to add Card's description, deadline, and comment feature has been addressed

Screenshots / Videos (front-end only):
Post at least one screenshot to show the feature in action. Only required in front-end work

Any information needed to test this feature (required):
add a MongoDB URI to the "MONGO_URI" key in an .env file on the servers' folder root directory. A MongoDB Atlas URI will be perfect for this
then add a Secret String Type Data to the "SECRET" key on the .env on the servers' folder root directory.
then run script below on your terminal from the server's folder root directory

```bash
$ npm run dev
```
Kindly find API Documentation for data format purposes
https://documenter.getpostman.com/view/10413337/UV5f6tSz

Any issues with the current functionality (optional):
List any problems where you were not able to complete all tasks on ticket
List any problems this PR may cause for the project or other active PRs
Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: server/models/board.js
Comment (ezeilo-su) [inline server/models/board.js:65]: Remove comment

File: server/models/board.js
Comment (ezeilo-su) [inline server/models/board.js:79]: Remove comment

File: server/models/board.js
Comment (ezeilo-su) [inline server/models/board.js:87]: This one too

File: server/models/card.js
Comment (ezeilo-su) [inline server/models/card.js:23]: Let get rid of this comment

File: server/models/card.js
Comment (ezeilo-su) [inline server/models/card.js:30]: Get rid of this one

File: server/models/card.js
Comment (ezeilo-su) [inline server/models/card.js:60]: Remove comment

File: server/models/column.js
Comment (ezeilo-su) [inline server/models/column.js:43]: Remove comment

File: server/models/column.js
Comment (ezeilo-su) [inline server/models/column.js:51]: Remove

File: server/models/column.js
Comment (ezeilo-su) [inline server/models/column.js:72]: Remove comment

File: server/models/column.js
Comment (ezeilo-su) [inline server/models/column.js:94]: Also remove this one

File: server/models/column.js
Comment (ezeilo-su) [inline server/models/column.js:102]: This one too

File: server/models/column.js
Comment (ezeilo-su) [inline server/models/column.js:121]: Remove comment

File: server/models/column.js
Comment (ezeilo-su) [inline server/models/column.js:129]: Remove comment

File: server/models/column.js
Comment (ezeilo-su) [inline server/models/column.js:146]: Remove comment

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
Quite a good work on this one :+1:
#### I have some comments for you
- So many file changes occurred due to your linters setup which is not consistent with the provided style guide. Please address this and make sure to apply proper formatting on your code, as I could not comment on each line of code in all cases.
- Your code is well written and nicely documented. Please remove unnecessary comments and console logs.
Please make the requested changes and wait for approval by a team member.
Happy coding!

**Comment by tejasj1718** [comment]
Hey Gideon,

I think you must merge this PR without waiting for any approval and merge the original issue. If you added another new issue into this PR then do not do that. You must open a new PR into main for that. Please clarify this with Ethan as well!

**Comment by gami34** [comment]
Dear Team,

All comments and console logs have been taken out.

we are still maintaining the same Linting

File: server/models/board.js
Comment (gami34) [reply server/models/board.js:44]: thanks allot for this

**Review by moffatethan — APPROVED** [review]
@gami34 Good changes! 🚀

**Pull request merged**
